### ISSUE TITLE
Error message caused by addition of line numbers

### ISSUE THREAD

**Issue by erboose** [End-to-end-provenance/DDG-Explorer#20]
The addition of source code line numbers for procedural nodes in ddg.txt causes DDG Explorer to display the error message "Unexpected tokens at end of line".

**Comment by blernermhc** [comment]
In the attached script, I see a different error message about Start and Finish nodes not matching.

I think that the best way to solve this is to include line numbers as attributes, like value is currently, rather than having RDataTracker place it in the node name.  DDGExplorer can still display it as part of the node name, or could give the user preferences on how they want to see it, whether part of the name, in a pop-up, etc.  Also, having it be a separate attribute rather than part of the name will make it easier for us to actually use it to look the line up in the script.

```R
data(mtcars)
cars.df <- mtcars

# Let's compute average MPG for 4 cylinder cars
# versus 8 cylinder cars.
ddg.start("Process 4 cylinder cars")
c4cars <- cars.df[cars.df$cyl == 4,]
m4 <- mean(c4cars$mpg)
ddg.finish("Process 4 cylinder cars")

ddg.start("Process 8 cylinder cars")
c8cars <- cars.df[cars.df$cyl == 8,]
m8 <- mean(c8cars$mpg)
ddg.finish("Process 8 cylinder cars")

# Let's create a warning node
if (m8 < m4) warning("Shocking! Eight cylinder cars let lower gas mileage!")

```

**Comment by erboose** [comment]
Agreed.  This is an easy change in RDataTracker once DDG Explorer has been modified to accept line numbers as a separate attribute.  Line numbers are already separate attributes in the JSON  file.

**Comment by blernermhc** [comment]
Fixed in DDG Explorer 2.06.
